playground/segment/server.js

```diff
@@ -10,7 +10,7 @@ app.get('/segment.html', function (req, res) {
 })
 
 app.get('/static/recorder.js', function (req, res) {
-    let filePath = path.join(__dirname, '/../../node_modules/rrweb/dist/rrweb.min.js')
```

Comment:
Doesn't look like they have a min version of the `js` file anymore

Reply:
yuck!

Reply:
well, less mean-ly i wonder what our minified file looks like now 🤔

Reply:
🤔 12kb bigger locally but doesn't look tooooo terrible